Content.Shared/_Starlight/Humanoid/StarlightHumanoidAppearanceSystem.cs

```diff
@@ -0,0 +1,32 @@
+using Content.Shared.Item;
+using Content.Shared.Starlight.Restrict;
+using Robust.Shared.Prototypes;
+
+// ReSharper disable CheckNamespace
+namespace Content.Shared.Humanoid;
+
+public sealed class StarlightHumanoidAppearanceSystem : EntitySystem
+{
+    private static readonly ProtoId<ItemSizePrototype> DefaultSize = "Huge";
+
+    [Dependency] private readonly SharedItemSystem _item = default!;
+
+    public void AddPickupData(EntityUid uid)
+    {
+        TryComp<HumanoidAppearanceComponent>(uid, out var humanoid);
+        if (humanoid != null)
+        {
+            EnsureComp<MultiHandedItemComponent>(uid);
+            EnsureComp<RestrictNestingItemComponent>(uid);
+
+            var item = EnsureComp<ItemComponent>(uid);
+
+            // Default values
+            var width = 6;
```

Comment:
same as above, you are using the same magic numbers but they aren't defined in a central location